File: app/src/main/java/com/taiko/noblenote/adapters/ViewHolder.java
package com.taiko.noblenote.adapters;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;

import rx.subscriptions.CompositeSubscription;

/**
 * Created by Taiko on 26.09.2016.
 */

public class ViewHolder extends RecyclerView.ViewHolder
{
    public CompositeSubscription mCompositeSubscription = new CompositeSubscription();

    public ViewHolder(View itemView)
    {
        super(itemView);
    }
}

File: app/src/main/java/de/blogspot/noblenoteandroid/adapters/ViewHolder.kt
package de.blogspot.noblenoteandroid.adapters

import androidx.databinding.ViewDataBinding
import androidx.recyclerview.widget.RecyclerView
import rx.subscriptions.CompositeSubscription

/**
 * Created by Taiko on 26.09.2016.
 */
class ViewHolder(val binding: ViewDataBinding) : RecyclerView.ViewHolder(binding.root) {
    // You might still use this for RxJava subscriptions if you don't use LiveData/Flow with binding
    val mCompositeSubscription = CompositeSubscription()
}